packages/react-native-client/src/stats/useRTCStatistics.ts

```diff
@@ -1,12 +1,8 @@
 import { takeRight } from 'lodash';
 import { useCallback, useEffect, useState } from 'react';
 
-import {
-  RTCInboundStats,
-  RTCOutboundStats,
-  RTCStats,
-} from '../RNFishjamClient.types';
-import RNFishjamClientModule from '../RNFishjamClientModule';
+import type { RTCInboundStats, RTCOutboundStats, RTCStats } from './types';
+import RNFishjamClientModule from '../FishjamClient';
```

Comment:
I do not like the way how we once use `RNFishjamClientModule` and `FishjamClient`. I think it should be unified.

Reply:
I will revert this change, and maybe do it in separate PR (as it would indeed require more changes

Reply:
👍 
Create ticket for it -> approve